file/src/main/scala/akka/stream/alpakka/file/model.scala

```diff
@@ -21,72 +21,127 @@ final class TarArchiveMetadata private (
     val filePathPrefix: Option[String],
     val filePathName: String,
     val size: Long,
-    val lastModification: Instant
+    val lastModification: Instant,
+    val linkIndicator: Byte
```

Comment:
Maybe rename it to `linkIndicatorByte` so we could later add more modelling for that value?

```
/* The linkflag defines the type of file */
#define  LF_OLDNORMAL '\0'       /* Normal disk file, Unix compatible */
#define  LF_NORMAL    '0'        /* Normal disk file */
#define  LF_LINK      '1'        /* Link to previously dumped file */
#define  LF_SYMLINK   '2'        /* Symbolic link */
#define  LF_CHR       '3'        /* Character special file */
#define  LF_BLK       '4'        /* Block special file */
#define  LF_DIR       '5'        /* Directory */
#define  LF_FIFO      '6'        /* FIFO special file */
#define  LF_CONTIG    '7'        /* Contiguous file */
```

Reply:
Ah, I see you have that table down below already.